index.dd

```diff
@@ -93,28 +93,36 @@ $(DIVC boxes,
                 Adam D. Ruppe's weekly newsletter. $(SPANC twid)
             )
         )
+        $(TOUR graduation-cap, Learn,
+            $(P Quick start with $(LINK2 http://tour.dlang.org,
+                Introduction to D), explore
```

Comment:
Is the name "Introduction to D" used anywhere else? The site itself seems to use "The Dlang Tour". How about "Take the [tour](http://tour.dlang.org),"?